ui/components/editor-submit-dialog.vue

```diff
@@ -5,56 +5,123 @@
     :shown="submit.state !== `closed`"
     :title="title">
 
+    <a
+      ref="downloadAnchorElement"
+      :href="downloadFileUrl"
+      :download="downloadFileName"
+      hidden />
```

Comment:
Is this still needed? Haven't we replaced this with `<app-download-button>`? If so, please delete:
* this tag
* `downloadFileUrl` and `downloadFileName` properties
* `fixtureJson` from ajax/submit-editor